org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
In the context of https://github.com/wildfly/wildfly/pull/9803 this is solely PRIVATE that's why I didn't add it here since IIRC we weren't going to be exposing PRIVATE capabilities here (especially since the plan is to change in the next release).